public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -207,83 +205,87 @@ include ../../../../_includes/_util-fns
 
 +makeExample('dependency-injection/ts/app/heroes/hero.service.1.ts',null, 'app/heroes/hero.service.ts' )
 :marked
-  Our `HeroService` exposes a `getHeroes()` method that returns
-  the same mock data as before but none of its consumers need to know that.
+  Our `HeroService` exposes a `getHeroes` method that returns
+  the same mock data as before, but none of its consumers need to know that.
 .l-sub-section
   :marked
     We aren't even pretending this is a real service.
     If we were actually getting data from a remote server, the API would have to be asynchronous,
     perhaps returning
-    [ES2015 promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)
+    [ES2015 promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
     We'd also have to rewrite the way components consume our service.
-    This is important but not important to our current story.
+    This is important in general, but not to our current story.
 :marked
   A service is nothing more than a class in Angular 2.
   It remains nothing more than a class until we register it with an Angular injector.
 
-  ### Configuring the Injector
+  ### Configuring the injector
 
   <a id="bootstrap"></a>
   We don't have to create an Angular injector.
   Angular creates an application-wide injector for us during the bootstrap process.
 
-+makeExample('dependency-injection/ts/app/main.ts', 'bootstrap', 'app/main.ts (bootstrap)')(format='.')
++makeExample('dependency-injection/ts/app/main.ts', 'bootstrap', 'app/main.ts (excerpt)')(format='.')
 :marked
-  We do have to configure the injector by registering the **providers** 
-  that create the services we need in our application.
+  We do have to configure the injector by registering the **providers**
+  that create the services that our application needs.
   We'll explain what [providers](#providers) are later in this chapter.
   Before we do, let's see an example of provider registration during bootstrapping:
-  
+  <!-- Again we show an example of bad code. Ew. I see we quickly explain
+  what you should do instead, but people might just copy the code and miss the explanation. -->
+
 +makeExample('dependency-injection/ts/app/main.1.ts', 'bootstrap')(format='.')
 :marked
   The injector now knows about our `HeroService`.
   An instance of our `HeroService` will be available for injection across our entire application.
 
   Of course we can't help wondering about that comment telling us not to do it this way.
-  It *will* work. It's just not a best practice. 
+  It *will* work. It's just not a best practice.
   The bootstrap provider option is intended for configuring and overriding Angular's own
-  pre-registered services.
-  
+  pre-registered services. <!-- such as what? -->
```

Comment:
"<!-- such as what? -->"  ... we don't want to go down that rabbit hole now. 

If you're curious, a good example is changing the router's default location strategy:
```
bootstrap(AppComponent, [
  ROUTER_PROVIDERS,
  provide(LocationStrategy,
         {useClass: HashLocationStrategy})
])
```